### DISCUSSION TITLE
Too many open files (os error 24) by ```quarto preview```

### DISCUSSION THREAD

**Discussion by LarsKarbach** [quarto-dev/quarto-cli#2233] [Q&A] [answered]
Hello, 

when i try to preview my blog in VS Code, i get ```ERROR: Too many open files (os error 24)```.  I use OpenSuse and my file limit  is 524288, which normally should be enough i guess? 

I would be very grateful if someone could help me further.

**Comment by mcanouil** [comment, marked as answer]
I don't think someone is likely to provide an answer without enough context.

- I assume, your blog is on GitHub, what's the repository?
- What version of Quarto are you using?
- Are you using the latest VSCode and Quarto for VSCode extension?
- Does the rendering work from CLI?
- How many actual files are opened in VSCode?
- Does the same happen from RStudio?
- etc.

**Comment by LarsKarbach** [comment]
Thanks for your effort. The problem has been fixed trough some changes in Linux.

**Comment by MichelDSchmid** [comment]
Hi to all

I had the same problem and could fix it with:
https://www.howtogeek.com/805629/too-many-open-files-linux/

regards
Michael
